### ISSUE TITLE
[Bug] Failing UI test - settingsScrollToBottomTest

### ISSUE THREAD

**Issue by rpappalax** [mozilla-mobile/fenix#4093]
The settingsScrollToBottomTest (UI test) is failing in Firebase on API 21.
Click on link to Firebase dashboard:
https://console.firebase.google.com/u/0/project/moz-fenix/testlab/histories/bh.66b7091e15d53d45/matrices/5480334586919337744/executions/bs.add80f09d43335b8
Drill down on failing API and click on tabs or "Test Results" button on upper right to view raw logs in GCP.
```
07-16 12:52:00.974: I/TestRunner(22842): ----- begin exception -----
07-16 12:52:00.976: I/TestRunner(22842): androidx.test.espresso.NoMatchingViewException: No views in hierarchy found matching: with id: org.mozilla.fenix.debug:id/switch_widget
```

**Comment by isabelrios** [comment]
This will be fixed as part of issue #4094. Please see comment there

**Comment by ekager** [comment]
Seems like a dupe, closing
